keps/sig-network/20200103-mixed-protocol-lb.md

```diff
@@ -405,6 +428,28 @@ In the first release:
   },
   "type": "object"
 },
+"io.k8s.api.core.v1.LoadBalancerStatus": {
+  "description": "LoadBalancerStatus represents the status of a load-balancer.",
+  "properties": {
+    "ingress": {
+      "description": "Ingress is a list containing ingress points for the load-balancer. Traffic intended for the servi  should be sent to these ingress points.",
+      "items": {
+        "$ref": "#/definitions/io.k8s.api.core.v1.LoadBalancerIngress"
+      },
+      "type": "array"
+    },
+    "conditions": {
```

Comment:
Generally conditions are just under Status since they represent potentially other facets of the resource.  I don't know if we have any precedent for nesting them further - do you think this is really a valuable trend to set?  It would be simpler and more consistent to define them higher at `status`.

Reply:
OK, updated. Let's see if I got it right now :)